pkg/command/service/generate.go

```diff
@@ -0,0 +1,200 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
+
+	knativeapis "knative.dev/pkg/apis"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
+	servingv1client "knative.dev/serving/pkg/client/clientset/versioned/typed/serving/v1"
+
+	"github.com/spf13/cobra"
+
+	"knative.dev/kperf/pkg"
+	"knative.dev/kperf/pkg/generator"
+)
+
+var (
+	count, interval, batch, concurrency, minScale, maxScale int
+	nsPrefix, nsRange, ns                                   string
+	svcNamePrefixDefault                                    string = "testksvc"
+	checkReady                                              bool
+	timeout                                                 time.Duration
+	ksvcClient                                              *servingv1client.ServingV1Client
+	err                                                     error
+)
```

Comment:
Do we need global variables?

Reply:
This part looks ugly to me and we're refining all the variable definitions, covered in #4. Will update later. Thanks!